packaging/torchvision/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
+channel_sources:
+  - defaults,conda-forge
```

Comment:
Do we need the `conda-forge` channel here? Can we remove it?

Reply:
Sure!